.github/workflows/extension-tests.yml

```diff
@@ -0,0 +1,60 @@
+name: Extension Tests
+
+on:
+  push:
+    branches: ["main"]
+  pull_request:
+    branches: ["**"]
+    paths:
+      - "extensions/**"
+      - ".github/workflows/extension-tests.yml"
+      # Exclude RISC-V test vectors
+      - "!extensions/rv32im/**"
```

Comment:
remove this: the RISC-V test vectors are in riscv.yml, whereas this workflow should cover rv32im-extension.yml